tests/ui_tools/test_boxes.py

```diff
@@ -208,7 +208,9 @@ def test_reset_search_text(self, panel_search_box):
     def test_keypress_ENTER(self, panel_search_box,
                             enter_key, log, expect_body_focus_set):
         size = (20,)
-        panel_search_box.panel_view.view.controller.editor_mode = True
+        panel_search_box.panel_view.view.controller.is_in_editor_mode = (
```

Comment:
The previous diff looked like it was setting the editor mode to True, which according to our helper functions should have called `enter_edit_mode_with()`. Did you use `is_in_editor_mode`  to makes things simpler? If that is the case, maybe adding a comment here would help.

Reply:
What do you mean by 'previous diff'? Do you mean what this commit replaces? I agree that setting it to `True` as it stands (without encapsulation) is literally the same as setting editor_mode to True via doing `enter_edit_mode_with`. However, that assumes we have all of the state in the controller available. Instead of mocking the controller state (as previously), these lambda methods are essentially mocking the controller behavior (on the assumption that it doesn't matter that `is_in_editor_mode` *always* returns `True` in this test).

Reply:
Re `previous diff`, I meant what this commit replaces. Yup. Thanks for clarifying. Re state: That was what I suspected as well. :)